i18n/zh/docusaurus-plugin-content-docs/version-2.5.json

```diff
@@ -96,27 +96,27 @@
     "description": "The label for category Checklist for Production-Ready Clusters in sidebar tutorialSidebar"
   },
   "sidebar.tutorialSidebar.category.Set Up Clusters from Hosted Kubernetes Providers": {
-    "message": "Set Up Clusters from Hosted Kubernetes Providers",
+    "message": "Setting up Clusters from Hosted Kubernetes Providers",
     "description": "The label for category Set Up Clusters from Hosted Kubernetes Providers in sidebar tutorialSidebar"
   },
   "sidebar.tutorialSidebar.category.Launch Kubernetes with Rancher": {
-    "message": "Launch Kubernetes with Rancher",
+    "message": "Launching Kubernetes with Rancher",
     "description": "The label for category Launch Kubernetes with Rancher in sidebar tutorialSidebar"
   },
   "sidebar.tutorialSidebar.category.Use New Nodes in an Infra Provider": {
-    "message": "Use New Nodes in an Infra Provider",
+    "message": "Launching New Nodes in an Infra Provider",
```

Comment:
The actual title of this page is so long that I compromised with a shorter version that at least starts with the same word as the title